nano/secure/blockstore.hpp

```diff
@@ -632,13 +632,28 @@ class frontier_store
 	virtual void for_each_par (std::function<void (nano::read_transaction const &, nano::store_iterator<nano::block_hash, nano::account>, nano::store_iterator<nano::block_hash, nano::account>)> const & action_a) const = 0;
 };
 
+class account_store
```

Comment:
Can you add some inline comments explaining a little what each of the methods of account_stores does?

Reply:
Added a simple comment there while I don't have a better one. After these break up PRs for each class I'm intending to do another one placing each class in a single file and starting a second round of code organization.